### DISCUSSION TITLE
Sortie de la version 3

### DISCUSSION THREAD

**Discussion by GravuTrad** [Kodi-vStream/venom-xbmc-addons#4282] [General]
Juste un petit post pour vous féliciter pour la sortie de cette excellente version 3.
Un changelog résumé est dispo quelque part?

**Comment by sizanic** [comment]
Merci.
pour les supports de communication, nous travaillons maintenant avec la communauté des alKODIques.

Une présentation de la V3 est à retrouver ici : https://lesalkodiques.com/voici-vstream-3-0/
